### DISCUSSION TITLE
Combobox has an autocomplete and autoComplete prop

### DISCUSSION THREAD

**Discussion by synnottd** [twilio-labs/paste#2781] [General] [answered]
I don't quite think it is a bug but it very, very annoying and may have taken me 3 hours to figure out. `autocomplete` is to make the Combobox autocomplete. `autoComplete` is to control the `autocomplete` attribute on a HTML `input` https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete.

**Comment by shleewhite** [comment]
Hi @synnottd! Thanks for letting us know about this. I am working on another ticket to clean up the Combobox types, so I can add this in. 

Will follow up when the PR is ready.

**Reply by shleewhite** [reply]
@synnottd Here is the PR if you want to follow it: https://github.com/twilio-labs/paste/pull/2782

**Comment by loreina** [comment, marked as answer]
Hi @synnottd, a PR was just merged to fix this issue: https://github.com/twilio-labs/paste/pull/2782. The fix will be available in our next version packages update: https://github.com/twilio-labs/paste/pull/2779. Thanks again for catching this!
